### PR TITLE
Seek related software landscape projects from search queries and expand project metric gathering

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds the capability to search GitHub using predefined queries which effects the projects dataset. The rough idea here was that the project data will likely change over time, so we can inform this with queries which may be performed periodically. It might also be that queries themselves could be estimated from the Cytomining ecosystem projects, but I wasn't sure about this yet.

Thanks in advance for any feedback you might have on this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,22 @@ poetry install
...
 Please follow installation steps above and then use a relevant Jupyter environment to open and explore the notebooks under the `notebooks` directory.
 
 ```bash
 # after creating poetry environment, run jupyter
 poetry run jupyter lab
 ```
+
+### Poe the Poet
+
+[Poe the Poet](https://poethepoet.natn.io/index.html) is used to define and run tasks defined within `pyproject.toml` under tables `[tool.poe.tasks*]`.
```
Comment (gwaybio) [inline README.md]:
```suggestion
We use [Poe the Poet](https://poethepoet.natn.io/index.html) to define and run tasks defined within `pyproject.toml` under the section `[tool.poe.tasks*]`.
```

Active voice. I also suggested a change to add clarity (which may not work)

File: README.md [hunk 2]
```diff
@@ -14,9 +14,22 @@ poetry install
...
+This allows for the definition and use of task workflow when implementing multiple procedures in sequence.
```
Comment (gwaybio) [inline README.md]:
```suggestion
This allows for the definition and use of a task workflow when implementing multiple procedures in sequence.
```

File: README.md [hunk 3]
```diff
@@ -14,9 +14,22 @@ poetry install
...
+Use the following to run, for example, the `cytomining_ecosystem` task:
```
Comment (gwaybio) [inline README.md]:
```suggestion

For example, use the following to run the `cytomining_ecosystem` task:
```

**Review by gwaybio — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (d33bs) [reply README.md]: Thanks! I'll commit this in.

File: README.md (hunk 2 above)
Comment (d33bs) [reply README.md]: Thanks! Will commit this in.

File: README.md (hunk 3 above)
Comment (d33bs) [reply README.md]: Thanks, this looks a lot better.

**Comment by d33bs** [comment]
Thank you @gwaybio for the review and suggestions, I've added these to the changes and will now merge.

**Pull request merged**